react/features/chat/functions.js

```diff
@@ -1,5 +1,49 @@
 // @flow
 
+import escapeRegexp from 'escape-string-regexp';
```

Comment:
This dependency is a grand total of 11 lines: https://github.com/sindresorhus/escape-string-regexp/blob/master/index.js Can we please just embed it here? Note in package-lock how other packages depend on an older version (1.x) so we'd be bundling it twice even.